.erb-lint.yml

```diff
@@ -1568,7 +1567,6 @@ linters:
           - select2-selection__clear
           - select2-selection__placeholder
           - select2-selection__rendered
-          - selected
```

Comment:
Having a greenstate of the linter, means that we can remove this from linter and consider it a "fresh css class"